topic-operator/src/main/java/io/strimzi/operator/topic/OperatorAssignedKafkaImpl.java

```diff
@@ -87,6 +81,9 @@ public OperatorAssignedKafkaImpl(AdminClient adminClient, Vertx vertx, Config co
 
     @Override
     public Future<Void> changeReplicationFactor(Topic topic) {
+
+        return Future.failedFuture("Changing replication factor is not supported");
+        /*
```

Comment:
What is the point of this huge comment? Do we plan to use this code later? Or sould it be just removed?